src/mirco_topologyutilities.h

```diff
@@ -49,14 +49,24 @@ namespace MIRCO
       int RandomGeneratorSeed, Teuchos::RCP<TopologyGeneration>& surfacegenerator);
 
   /**
-   * @brief Compute the maximum height (zmax) and the mean height (zmean) of the topology.
+   * @brief Store maximum and mean height of the topology
+   *
+   * @param max_ Maximum height of the topology
+   * @param mean_ Mean height of the topology
```

Comment:
In doxygen `@param` refers to the parameters of a function. In this case we mean member variables of the structure. You can either document them below, or don't document them individually as the naming is quite clear in this case.